pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[tool.poetry]
+name = "simple-settings"
+version = "1.1.0"
+description = "A simple way to manage your project settings."
+authors = ["Diego Garcia <drgarcia1986@gmail.com>"]
+license = "MIT"
+keywords=["django", "flask", "bottle", "tornado", "settings", "configuration", "conf"]
+classifiers=[
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 3',
+]
+packages = [
+    { include = "simple_settings" },
+    { include = "strategies", from = "simple_settings" },
+    { include = "dynamic_settings", from = "simple_settings" },
+]
+repository = "https://github.com/drgarcia1986/simple-settings"
+
+[tool.poetry.urls]
+"Download" = "https://github.com/drgarcia1986/simple-settings/tarball/master"
+
+[tool.poetry.dependencies]
+python=">=3.6.1,<4.0"
+jsonpickle = { version = "2.0.0", optional = true }
+toml = { version = "0.10.2", optional = true }
+PyYAML = { version = "6.0", optional = true }
+SQLAlchemy = { version = "1.4.26", optional = true }
+pymemcache = { version = "3.5.0", optional = true }
+six = { version = "1.16.0", optional = true}
+redis = { version = "3.5.3", optional = true }
+boto3 = { version = "1.9.201", optional = true }
+
+
+[tool.poetry.dev-dependencies]
+bumpversion="0.6.0"
+flake8="4.0.1"
+isort="5.9.3"
+moto="2.3.0"
```

Comment:
The version bump to `moto` and the removal of `consulate` were committed only to `pyproject.toml`, but not to `poetry.lock`.

The `poetry.lock` hash is out of date and running `poetry.install` results in the following message:

```
Installing dependencies from lock file
Warning: The lock file is not up to date with the latest changes in pyproject.toml. You may be getting outdated dependencies. Run update to update them.

  SolverProblemError

  Because simple-settings depends on moto (2.3.0) which doesn't match any versions, version solving failed.
[...]
```

Reply:
whops, you're right! I'll commit the updated lock file